src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -444,7 +445,7 @@ public ItemStack decrStackSize(final int index, final int count)
             tempIndex -= nonnulllist.size();
         }
 
-        return list != null && !(list.get(tempIndex)).isEmpty() ? ItemStackHelper.getAndSplit(list, tempIndex, count) : ItemStack.EMPTY;
+        return list != null && !(list.get(tempIndex)).isEmpty() ? ItemStackHelper.getAndSplit(list, tempIndex, count) : ItemStackUtils.EMPTY;
```

Comment:
Good catch, will fix those.
I am not too worry if I miss some because at one pount I will be comparing 1.11 and 1.10 and make sure that the code is identical